website/content/snippets/heron-on-kubernetes-config.md

```diff
@@ -0,0 +1,175 @@
+config file. For k8s deployments, Heron can use the `S3Uploader`.
+Details on configuring the S3 Uploader can be found in the documentation [here](../../uploaders/s3).
+
+## Kubernetes Scheduler Configuration
+
+To configure Heron to use the Kubernetes scheduler, modify the `scheduler.yaml`
+config file specific for the Heron cluster. The following must be specified
+for each cluster:
+
+* `heron.class.scheduler` --- Indicates the class to be loaded for Kubernetes scheduler.
+You should set this to `com.twitter.heron.scheduler.kubernetes.KubernetesScheduler`
+
+* `heron.class.launcher` --- Specifies the class to be loaded for launching and
+submitting topologies. To configure the Kubernetes launcher, set this to
+`com.twitter.heron.scheduler.kubernetes.KubernetesLauncher`
+
+* `heron.directory.sandbox.java.home` --- Specifies the location of the Java 8 JRE. Depending on 
+the Docker container you're using for executor image, this may be different. If you're using
+the Streaml.io-hosted containter, this will already be available within the environment as 
+`$JAVA_HOME`
+
+* `heron.directory.home` --- Specifies the directory for the core binaries. This should be set to
+`$HERON_HOME`
+
+* `heron.director.conf` --- Specifies the directory where the Heron configuration files are located.
+This should be set to `/opt/heron/heron-conf/`
+
+* `heron.kubernetes.scheduler.uri` --- Specifies the URI for the Kubernetes API service. If you're 
+using `kubectl proxy` to access your API, this will need to be set to `http://localhost:8001` if 
+you're using the standard proxy configuration.
+
+* `heron.scheduler.is.service` --- This config indicates whether the scheduler
+is a service. In the case of Kubernetes, it should be set to `False`.
+
+* `heron.executor.docker.image` --- Specified the Docker image to be used for running the executor.
+To use the Docker image created for the latest release, you can set this value to 
+`streamlio/heron:<version>-<os>`. Example: `streamlio/heron:0.14.8-ubuntu14.04`
+
+### Example Kubernetes Scheduler Configuration
+
+```yaml
+# scheduler class for distributing the topology for execution
+heron.class.scheduler:                       com.twitter.heron.scheduler.kubernetes.KubernetesScheduler
+
+# launcher class for submitting and launching the topology
+heron.class.launcher:                        com.twitter.heron.scheduler.kubernetes.KubernetesLauncher
+
+# location of java - pick it up from shell environment
+heron.directory.sandbox.java.home:          $JAVA_HOME
+
+heron.directory.conf:                       "/opt/heron/heron-conf/"
+
+heron.directory.home:                       $HERON_HOME
+
+# The URI of kubernetes API
+heron.kubernetes.scheduler.uri: "http://localhost:8001"
+
+# Invoke the IScheduler as a library directly
+heron.scheduler.is.service:                  False
+
+# docker repo for executor
+heron.executor.docker.image: 'streamlio/heron:0.14.8-ubuntu14.04'
```

Comment:
Let's update this to a heron image heron/heron:latest

Reply:
This is just a WIP for now. I'll ping you when it's ready for review.